### PR TITLE
Add preferreduilang to F# compiler options and FSI options table

### LINKED ISSUE #4183: Add prefereduilang option to the F# compiler options page
ref: https://github.com/Microsoft/visualfsharp/issues/4243#issuecomment-359881759

@KevinRansom said:

```
use: fsc --preferreduilang:us-EN

in projects add $(OtherFlags);--preferreduilang:us-EN

There is a bug, it doesn't seem to set it for FSI, and that needs to be fixed.

fsc.exe" --preferreduilang:de-DE
Microsoft (R) F# Compiler version 10.1.0 for F# 4.1
Copyright (c) Microsoft Corporation. Alle Rechte vorbehalten.

error FS0207: Keine Eingaben angegeben.

fsc.exe" --preferreduilang:us-EN
Microsoft (R) F# Compiler version 10.1.0 for F# 4.1
Copyright (c) Microsoft Corporation. All Rights Reserved.

error FS0207: No inputs specified
```

This should be a fairly simple addition to the table.

### PR DESCRIPTION
**Pull request by cartermp**
Fixes #4183

Also moves the FSI options table to the language reference, since that's a better place based on the fact that the compiler options table is there, too.

### CODE REVIEW

**Review by Roadkitten — APPROVED** [review]

**Review by BillWagner — APPROVED** [review]
The change LGTM.  I'll close and reopen to trigger a new build as the OPS build seems stuck.

Once the build is clean, I'll :shipit:

**Comment by cartermp** [comment]
Hang on - I'll add this to F# Interactive as well

**Comment by mairaw** [comment]
there are some merge conflicts on the PR @cartermp

File: docs/fsharp/language-reference/compiler-options.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ The following table shows compiler options listed alphabetically. Some of the F#
...
 |**--pdb:&lt;pdb-filename&gt;**|Names the output debug PDB (program database) file. This option only applies when **--debug** is also enabled.<br /><br />This compiler option is equivalent to the C# compiler option of the same name. For more information, see [&#47;pdb &#40;C&#35; Compiler Options&#41;](https://msdn.microsoft.com/library/ms228625.aspx).|
 |**--platform:&lt;platform-name&gt;**|Specifies that the generated code will only run on the specified platform (**x86**, **Itanium**, or **x64**), or, if the platform-name **anycpu** is chosen, specifies that the generated code can run on any platform.<br /><br />This compiler option is equivalent to the C# compiler option of the same name. For more information, see [&#47;platform &#40;C&#35; Compiler Options&#41;](https://msdn.microsoft.com/library/zekwfyz4.aspx).|
+|**--preferreduilang:&lt;lang&gt;**| Specifies the preferred output language culture name (e.g. es-ES, ja-JP). |
```
Comment (mairaw) [inline docs/fsharp/language-reference/compiler-options.md]: e.g. -> for example,

File: docs/fsharp/language-reference/fsharp-interactive-options.md [hunk 2]
```diff
@@ -52,6 +52,7 @@ Where lists appear in F# Interactive option arguments, list elements are separat
 |**--nologo**|Same as the **fsc.exe** compiler option. For more information, see [Compiler Options](../../language-reference/compiler-options.md).|
 |**--nowarn:&lt;warning-list&gt;**|Same as the **fsc.exe** compiler option. For more information, see [Compiler Options](../../language-reference/compiler-options.md).|
 |**--optimize**[**+**&#124;**-**]|Same as the **fsc.exe** compiler option. For more information, see [Compiler Options](../../language-reference/compiler-options.md).|
+|**--preferreduilang:&lt;lang&gt;**| Specifies the preferred output language culture name (e.g. es-ES, ja-JP). |
```
Comment (mairaw) [inline docs/fsharp/language-reference/fsharp-interactive-options.md]: ditto

File: docs/fsharp/tutorials/fsharp-interactive/index.md [hunk 3]
```diff
@@ -94,5 +94,5 @@ test
 
 |Title|Description|
 |-----|-----------|
-|[F# Interactive Options](fsharp-interactive-options.md)|Describes command line syntax and options for the F# Interactive, fsi.exe.|
+|[F# Interactive Options](../../language-reference/fsharp-interactive-options.md)|Describes command line syntax and options for the F# Interactive, fsi.exe.|
```
Comment (mairaw) [inline docs/fsharp/tutorials/fsharp-interactive/index.md]: command line -> command-line

**Review by mairaw — CHANGES_REQUESTED** [review]
some small nits to be fixed

**Review by mairaw — APPROVED** [review]
LGTM. You can merge it when build finishes

**Comment by cartermp** [comment]
🎉

**Comment by cartermp** [comment]
öööf that build is taking a while

**Comment by mairaw** [comment]
The webhook seems to be failing for this PR. Not sure why. I can open a LSI in the morning if the issue persists.

**Comment by JRAlexander** [comment]
Build still hung. Closing and reopening.

**Comment by JRAlexander** [comment]
Submitting LSI

**Pull request merged**